tests/rules/test_drmemory_no_app_specified.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from thefuck.rules.drmemory_no_dashdash import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command('drmemory a.out', 'Usage: drmemory [options] --'),
+    Command('/etc/bin/drmemory test.exe', 'Usage: drmemory [options] --'),
```

Comment:
Here, only `script` and `output` could be parametrized. Then in the function body `Command` can be then instantiated as in `assert match(Command(script, output))`.

Nit: Also, to reduce repetition a bit, a fixture `output` could be used.

Check [tests/rules/test_git_push.py](https://github.com/nvbn/thefuck/blob/master/tests/rules/test_git_push.py) for an example of both cases.